src/Nox.Reference.Data.World/Migrations/WorldDbContextModelSnapshot.cs

```diff
@@ -0,0 +1,236 @@
+﻿// <auto-generated />
+using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+using Nox.Reference.Data.World;
+
+#nullable disable
+
+namespace Nox.Reference.Data.World.Migrations
+{
+    [DbContext(typeof(WorldDbContext))]
+    partial class WorldDbContextModelSnapshot : ModelSnapshot
+    {
+        protected override void BuildModel(ModelBuilder modelBuilder)
+        {
+#pragma warning disable 612, 618
+            modelBuilder.HasAnnotation("ProductVersion", "7.0.4");
+
+            modelBuilder.Entity("Nox.Reference.Data.World.Currency", b =>
+                {
+                    b.Property<int>("Id")
+                        .ValueGeneratedOnAdd()
+                        .HasColumnType("INTEGER");
+
+                    b.Property<int>("BanknotesId")
+                        .HasColumnType("INTEGER");
+
+                    b.Property<int>("CoinsId")
+                        .HasColumnType("INTEGER");
+
+                    b.Property<int>("DecimalDigits")
+                        .HasColumnType("INTEGER");
+
+                    b.Property<string>("DecimalSeparator")
+                        .IsRequired()
+                        .HasColumnType("TEXT");
```

Comment:
Should we restrict the length of such fields?
Or at least add todo/create an issue

Reply:
#19